### PR TITLE
Wrapped sech, csch, asech and acsch functions

### PR DESCRIPTION
**Pull request by ShikharJ**

### CODE REVIEW

**Comment by ShikharJ** [comment]
@isuruf Any idea why is this error coming up? `acsch` and `asech` are defined in sympy.
```
AttributeError: module 'sympy' has no attribute 'acsch'
```

**Comment by isuruf** [comment]
Are they defined in the version we are testing with?

**Comment by ShikharJ** [comment]
@isuruf Not sure. How can I check for the `SymPy` version we are using (and possibly update it)?

**Comment by ShikharJ** [comment]
Ping @isuruf.

File: symengine/lib/symengine_wrapper.pyx [hunk 1]
```diff
@@ -216,6 +230,11 @@ def sympy2symengine(a, raise_error=False):
         return DenseMatrix(row, col, v)
     elif isinstance(a, sympy.polys.domains.modularinteger.ModularInteger):
         return PyNumber(a, sympy_module)
+    elif sympy.__version__ > sympy_version:
```
Comment (isuruf) [inline symengine/lib/symengine_wrapper.pyx]: Can you replace `sympy_version` with `1.0`? Having it as a global variable makes no sense

**Comment by ShikharJ** [comment]
Ping @isuruf.

**Comment by isuruf** [comment]
Thanks

**Pull request merged**